README.md

```diff
@@ -747,7 +747,7 @@ The table below shows the supported package managers and their support level in
 
 Tool     | Version |
 ---      |---------|
-Go*      | 1.20.7, 1.21.0  |
+Go*      | 1.20.7, 1.22.0 (no workspace vendoring support), 1.23.0  |
```

Comment:
Nitpick: I wonder if we need to specify 1.22. The second version in this table refers to the maximum supported version, so I'd just keep it as 1.23.
```suggestion
Go*      | 1.20.7, 1.23.0 (no workspace vendoring support)|
```

Reply:
Uhm, we don't, but I figured if we need to mention limitations better tie it with the version that they apply to, but it might be just me who'd be confused what particular feature that is and which version introduced it. If you don't think it's valuable I can drop it as you suggest.